target/linux/ramips/dts/ZBT-WG3526.dtsi

```diff
@@ -0,0 +1,118 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mediatek,mt7621-eval-board", "mediatek,mt7621-soc";
```

Comment:
I saw the mediatek,mt7621-eval-board used a few times. Does it really does something or is it just copied over and over again?